optimization/optimizer/matRad_OptimizerIPOPT.m

```diff
@@ -138,16 +153,22 @@
                             fprintf('Press q to terminate the optimization...\n');
                             qCallbackSet = true;
                         catch
-                            fprintf('Manual optimization termination with q disabled.\n');
+                            matRad_cfg.dispInfo('Manual termination with q not possible due to failing callback setup.\n');
                         end
                 end                
             end
             
             %ipoptStruct.options = obj.options;
             obj.abortRequested = false;
             obj.plotFailed = false;
+            
             % Run IPOPT.
-            [obj.wResult, obj.resultInfo] = ipopt(w0,funcs,ipoptStruct);
+            try
+                [obj.wResult, obj.resultInfo] = ipopt(w0,funcs,ipoptStruct);
+            catch ME
+                errorString = [ME.message '\nThis error was thrown by the MEX-interface of IPOPT.\nMex interfaces can raise compatability issues which may be resolved by compiling them by hand directly on your particular system.'];
```

Comment:
Weltidee!! :)

Reply:
Because we tell people that mex files can be an issue? :-D